### DISCUSSION TITLE
About measuring latent variable via psychological scales

### DISCUSSION THREAD

**Discussion by minhengw** [jazznbass/scan#78] [Q&A] [answered]
I just found the single-case design and relevant quantitative methods VERY POWERFUL! Particularly, when trying to evaluate the effectiveness of an intervention or a new program, many times there are only a few subjects (very small sample, usually n < 10), which means any parametric or conventional statistical methods or tests are not suitable. That's why I am so excited about the content in the website:
https://jazznbass.github.io/scan-Book/ 

However, I have a few questions below, and hope to know more about how to implement the single-case analytics (scan):

1. Do you recommend measuring latent construct (via psychological scale) and using scan to perform the quantitative analysis (i.e. statistical test, like randomization test)? Because I am not sure whether "frequently" measuring the same construct is appropriate for subjects.

2. Could you recommend any research papers or empirical research using "scan"? I am very happy to read them and know more about the contexts of using this tool.

Thank you so much!

**Comment by researchtool** [comment, marked as answer]
Thank you for the praise and your interest in the scan package. :)

The basic idea in single case design research is, of course, different (i.e., multiple data points instead of many individuals). But if you take a closer look, the boundaries are of course fluid and you can also see this in the latest methodological developments.

We have recently published a paper with power analyses, and there are recent papers by other teams on this question, too. However, the number of recommended measurement time points depends on many factors (strength of the level and slope effect, strength of the baseline trend, baseline level, and variability of the DV) as well as the planned evaluation method.
Since the DV should be measured with equally difficult and content-valid items, I would be rather cautious about using a psychological scale. It is common to use simple behavioral observations that are ecologically valid and accurately reflect the target behavior or competency. however, the question can, of course, only be answered in the context of the research question and the operationalization of the DV in question.

https://www.frontiersin.org/articles/10.3389/feduc.2022.917944/full

I'll leave you with the task to assess the quality ;-P, but here are some papers using scan for their analyses (some of them provide scripts via supplement or osf):
https://www.mdpi.com/2077-0383/11/15/4553/htm
https://iejee.com/index.php/IEJEE/article/view/1773/578
https://doi.org/10.1111/papt.12390

Hope that helps. :)
timo

**Reply by minhengw** [reply]
Dear Timo,

Thank you for your prompt reply! It seems that there are still some measures using psychological scale (such as Big-Five), but the use of scale should be cautious.

I will carefully read the papers provided, thank you so much!

Sincerely
David

**Comment by minhengw** [comment]
Dear Timo,

Thank you for your prompt reply! It seems that there are still some
measures using psychological scale (such as Big-Five), but the use of
scale should be cautious.

I will carefully read the papers provided, thank you so much!

Sincerely
David
